### PR TITLE
Add SELinux support

### PR DESCRIPTION
**Pull request by TommyTran732**
Since the staples are created in tmpfs, they will have the user_tmp_t context which will break nginx when it tries to reload. This fixes the problem assuming the policies set the output dir to be labeled as httpd_config_t

### CODE REVIEW

**Comment by TommyTran732** [comment]
Not sure why the linter is not happy with what I wrote but I see nothing wrong with it

**Comment by wj25czxj47bu6q** [comment]
@TommyTran732 Consider using `mv -Z` instead of `restorecon`.

**Comment by TommyTran732** [comment]
Done :)

**Review by tomwassenberg — COMMENTED** [review]
Thanks for the PR! Left a minor remark, will merge right after that's addressed :)

**Comment by TommyTran732** [comment]
Done :)

**Review by tomwassenberg — APPROVED** [review]

**Pull request merged**
